src/pageContainer.component.tsx

```diff
@@ -19,6 +20,26 @@ const PageContainer = (): React.ReactElement => {
   const theme = useTheme();
   const isViewportMdOrLarger = useMediaQuery(theme.breakpoints.up('md'));
 
+  React.useEffect(() => {
+    const clearToasts = (): void => {
+      toastr.clean();
```

Comment:
Is it not possible to mock `toastr` with `jest.mock`, and then check this is called to cover these lines?

Reply:
No, all of the configuration of the toasts (listening to the plugins) is done in the app.tsx